pkg/services/teamgroupsync/teamgroupsync.go

```diff
@@ -0,0 +1,24 @@
+package teamgroupsync
```

Comment:
```suggestion
// Package teamgroupsync provides the interface for the Grafana Enterprise team sync feature.
package teamgroupsync
```

Reply:
Great! Thank you. Totally forgot about package comments.